object.js

```diff
@@ -27,10 +27,17 @@ function hint(gj, options) {
                 line: _.__line__
             });
         } else if (!types[_.type]) {
-            errors.push({
-                message: 'The type ' + _.type + ' is unknown',
-                line: _.__line__
-            });
+            if (typesLower.indexOf(_.type.toLowerCase()) > -1) {
```

Comment:
[1.4](https://circle-artifacts.com/gh/geojson/draft-geojson/77/artifacts/0/home/ubuntu/draft-geojson/draft.html#rfc.section.1.4) specifies that geojson types and geometry types are case sensitve. But, yes, the current error message is not useful without suggesting the corrected case. I'll make the change.